clang/tools/clang-format/clang-format.el

```diff
@@ -205,14 +288,60 @@ uses the function `buffer-file-name'."
       (delete-file temp-file)
       (when (buffer-name temp-buffer) (kill-buffer temp-buffer)))))
 
+;;;###autoload
+(defun clang-format-git-diffs (&optional style assume-file-name)
```

Comment:
A more general point, would it make sense to call this `clang-format-vc-diffs` - which currently only supports git?

This way we wouldn't need `clang-format-hg-diffs` or separate commands to support other version control systems in the future.

Reply:
I think the term diff would be more consistent with existing emacs functions e.g. vc-diff, diff-delete-trailing-whitespace etc.

Reply:
The reason I chose `diffs` was to indicate it wasn't just the diff at point or something, but Ill rename `clang-format-vc-diff`